tree/treeplayer/inc/TBranchProxy.h

```diff
@@ -116,20 +121,26 @@ namespace Detail {
 
       void Reset();
 
+      Bool_t Notify() {
+         return Setup();
+      }
+
       Bool_t Setup();
 
       Bool_t IsInitialized() {
-         return fLastTree && fCurrentTreeNumber == fDirector->GetTree()->GetTreeNumber() && fLastTree == fDirector->GetTree();
+         return fInitialized;
+         // return fLastTree && fCurrentTreeNumber == fDirector->GetTree()->GetTreeNumber() && fLastTree == fDirector->GetTree();
```

Comment:
Is this worth keeping?

Reply:
no.  WIP .... last step is to remove fLastTree completely.

Reply:
removed